lambdas/functions/data_transfer_manager/data_transfer_manager.py

```diff
@@ -281,11 +294,14 @@ def construct_directory_from_flagship_and_submission(flagship, submission):
     return f'{flagship}/{submission}/'
 
 
-def create_cli_s3_object_batch_job(source_bucket_name, source_s3_key, cli_op: str = 'mv'):
+def create_cli_s3_object_batch_job(source_bucket_name, source_s3_key, target_s3_key=None, cli_op: str = 'mv'):
```

Comment:
Why is the source bucket configurable, but not the destination bucket?
Is that so it can be used to copy from staging and/or result into store?

Reply:
Right. All good.
If it's just a well defined local use case, it's fine. 

I was concerned about general usage, where buckets change and submission directories for source and target don't match. In the current case that should not be an issue.